### DISCUSSION TITLE
SO 2.4 Playbook -

### DISCUSSION THREAD

**Discussion by EchoGangster** [Security-Onion-Solutions/securityonion#11087] [2.4] [answered]
In Playbook all plays are missing.  There is nothing in active or inactive plays.  
<img width="1440" alt="Screenshot 2023-08-17 at 8 18 17 PM" src="https://github.com/Security-Onion-Solutions/securityonion/assets/34453426/4b526d1f-4b59-441a-bd28-f6fb38ad7c8a">

**Comment by defensivedepth** [comment]
Please read the following specifically the `Please include the following details where you can`: and update your post - 

 Read before posting! https://github.com/Security-Onion-Solutions/securityonion/discussions/1720

**Reply by EchoGangster** [reply]
SO 2.4.10 upgraded from the previous version.  Installed using the ISO. Running on VMWare ESXI 7.03.  
The Playbook off of the SOC page does not have any plays loaded in the playbook as shown in the picture above.

**Reply by defensivedepth** [reply]
What version did you upgrade from?

**Comment by EchoGangster** [comment]
Whatever the last Beta version was before the GA release.   I have attempted to run so-playbook-update and there are errors there.  I am waiting for it to finish before checking opt/so/rules/elastalert/ to see if plays are there.   Below is the output from so-playbook-update

`sigma/rules/windows/powershell/powershell_script/posh_ps_script_with_upload_capabilities.yml
No Current Play - Create New Play in Playbook
{'play_creation': 401, 'play_url': 'failed'}
new


sigma/rules/windows/powershell/powershell_script/posh_ps_security_software_discovery.yml
No Current Play - Create New Play in Playbook
Play Creation Error - 
Play Creation Error - 
Play Creation Error - `

**Comment by EchoGangster** [comment]
Check the /opt/so/rules/elastalert/playbook directory and there 0 rules in there.  

`sudo so-playbook-sigma-refresh --force
Refreshing Sigma & regenerating plays... 

-= Started: 2023-08-18 16:45:49.400817-=


-= Parsed Playbook Plays: 0 -=

rm: cannot remove '/opt/so/rules/elastalert/playbook/*.yaml': No such file or directory

-= Started: 2023-08-18 16:45:49.807581-=


-= Parsed Playbook Plays: 0 -=



-= Maintenance Summary =-

Active Plays: 0
-----------------
Missing ElastAlert Configs: 0
Inactive Plays: 0
-----------------
Out of Sync ElastAlert Configs: 0
`

**Comment by EchoGangster** [comment]
Looks like the file path pointing to the GitHub directory for the Sigma rules is wrong. 

`-= Parsing current Plays in Playbook -=



-= Parsed Playbook Plays: 0 -=



sigma/rules/windows/builtin/win_alert_mimikatz_keywords.yml
No Current Play - Create New Play in Playbook
{'play_creation': 401, 'play_url': 'failed'}
new


sigma/rules/windows/builtin/application/Other/win_av_relevant_match.yml
No Current Play - Create New Play in Playbook
{'play_creation': 401, 'play_url': 'failed'}`

<img width="1440" alt="Screenshot 2023-08-18 at 7 01 55 PM" src="https://github.com/Security-Onion-Solutions/securityonion/assets/34453426/3a6eee72-4404-4d4a-9d25-522de512c30f">

**Comment by defensivedepth** [comment, marked as answer]
Please try `so-playbook-reset` - this will take a while to complete.

**Comment by EchoGangster** [comment]
@defensivedepth Thank you for your help!  Looks like so-playbook-reset solved the issue.
